browser/net/brave_request_handler.h

```diff
@@ -47,8 +47,6 @@ class BraveRequestHandler {
 
  private:
   void SetupCallbacks();
-  void InitPrefChangeRegistrar();
-  void OnReferralHeadersChanged();
   void OnPreferenceChanged(const std::string& pref_name);
```

Comment:
unrelated to this PR, but `OnPreferenceChanged` and `UpdateAdBlockFromPref` do not exist in .cc so also can be erased :)